cli/gluster-block.c

```diff
@@ -640,6 +640,8 @@ glusterBlockCreate(int argcount, char **options, int json)
   cobj.json_resp = json;
   cobj.mpath = 1;
   cobj.prealloc = 1;
+  /* default 64 MiB in kernel space */
+  cobj.rb_size = 64;
```

Comment:
Ok, what about the block-size default then ?

Reply:
Wait, there will be a problem that when the dyn ring buffer is supported, it won't be 64MiB, so the 64MiB as default is incorrect.

How about set it to -1, which means as default ? And the same for the block-size....

Reply:
size in negative looks a bit wierd for me, instead may be say,
```
BLKSIZE: default
RINGBUFSIZE: default
```

Reply:
Yeah, this also makes sense.
Let's split this into another PR , there needs much more code to adapt it and which will be conflicting with the big patch followed.